models/games-model.js

```diff
@@ -11,3 +11,26 @@ exports.selectCategories = () => {
       return categories.rows;
     });
 };
+
+exports.selectReviews = () => {
+  return db
+    .query(
+      `
+    SELECT users.username, reviews.owner, reviews.title, reviews.review_id, reviews.category, reviews.review_img_url, reviews.created_at, reviews.votes, reviews.designer, COUNT(comments.review_id)::int AS comment_count 
+    FROM reviews
+    LEFT JOIN users ON reviews.owner = users.username
```

Comment:
this isn't needed when the username information is already present on the review object